LibraryAssign.java

```diff
@@ -0,0 +1,49 @@
+package libraryassign;
+
+import java.util.Date;
+import java.util.Scanner;
+
+/**
+ *
+ * @author Shaksham Kapoor
+ */
+public class LibraryAssign 
+{
+    static LibraryFunc libf  = new LibraryFunc();
+    
+    public static void main(String[] args) 
+    {
+        Books book = new Books("Intro. to Statistical Learning", "Freedie and Hestie", 978146147, 440, new Date());
+        Books book1 = new Books("Elements of Statistical Learning", "Freedie and Hestie", 978146148, 1020, new Date());
+        libf.addBook(book);
+        libf.addBook(book1);
+        
+        System.out.println("List of available books in the library : ");
+        viewLibrary();
+    }
+    
+    public static void viewLibrary()
+    {

```

Comment:
So far you've only identified one object which is book, I count 3 objects in total.